.travis.yml

```diff
@@ -28,6 +26,8 @@ addons:
       - libglew-dev
       - libopenni-dev
       - python3-pip
+      - python3-sphinxcontrib.doxylink
```

Comment:
Wow, just found that Ubuntu only added this in artful, even though it's in Debian since 2016.. So you have to go back to pip, sorry.